.changeset/mighty-spoons-worry.md

```diff
@@ -0,0 +1,8 @@
+---
+'@keystonejs/adapter-knex': major
+'@keystonejs/adapter-mongoose': major
```

Comment:
```suggestion
```
These aren't necessary here, they're in the PR making the id field non-nullable for a specific reason that isn't relevant here